releasenotes/notes/ga-runtime-metrics-4fba9b4dbe63ce0f.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
```

Comment:
Can the configuration.rst file also be updated with the change: https://github.com/DataDog/dd-trace-py/blob/f76498f23b02b4cf40cade96db9f8870bbba703a/docs/configuration.rst?plain=1#L854 ?